From 9a06ec080ba96c8694eb307c390392f3f1bdba10 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Wed, 30 Mar 2022 15:21:27 -0700
Subject: [PATCH] Update to C++17

Needed to fix Qt error on macOS:

  In file included from /usr/local/include/QtCore/qnamespace.h:44:
  /usr/local/include/QtCore/qglobal.h:96:6: error: "Qt requires a C++17 compiler"
  #    error "Qt requires a C++17 compiler"
       ^
---
 rviz_rendering/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rviz_rendering/CMakeLists.txt b/rviz_rendering/CMakeLists.txt
index 53786f8c..6f038130 100644
--- a/rviz_rendering/CMakeLists.txt
+++ b/rviz_rendering/CMakeLists.txt
@@ -2,9 +2,9 @@ cmake_minimum_required(VERSION 3.5)
 
 project(rviz_rendering)
 
-# Default to C++14
+# Default to C++17
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 17)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-- 
2.32.0 (Apple Git-132)

